Render.py

```diff
@@ -231,7 +237,9 @@ def setProperties(self,obj):
             obj.addProperty("App::PropertyFile","OutputImage","Render", QT_TRANSLATE_NOOP("App::Property","The image saved by this render"))
         if not "OpenAfterRender" in obj.PropertiesList:
             obj.addProperty("App::PropertyBool","OpenAfterRender","Render", QT_TRANSLATE_NOOP("App::Property","If true, the rendered image is opened in FreeCAD after the rendering is finished"))
-            obj.GroundPlane = False
```

Comment:
It seems to me better to leave the groundplane off by default, no?

Reply:
I have no idea why i removed this line. I'm sure it was not on purpose.

I will have a look what happened here and fix it

Reply:
Now I know again what this commit was goot for. In this line, the wrong Property was initialized. Because "OpenAfterRender" is added, when it is missing, but "GroundPlane" is set to false in this case.
But the GroundPlane is initialized a few lines before to false anyways.

```python
if not "GroundPlane" in obj.PropertiesList:
            obj.addProperty("App::PropertyBool","GroundPlane","Render", QT_TRANSLATE_NOOP("App::Property","If true, a default ground plane will be added to the scene"))
            obj.GroundPlane = False
```

So this should not have any effect on the default GroundPlane value.